### PR TITLE
[DQL2-26] QA Remediation

### PR DESCRIPTION
**Pull request by salsa-nathan**
- Adjusted `snippets/comment_deleted.html` to consider `show_deleter` param when displaying deleted comment message.
- Adjusted base `comments_list.html` to always display deleter's username by default.
- Default `show_deleter` to True - can be overridden by theme template.

@duttonw and @ThrawnCA - for review

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ flake8==3.8.2
 nose==1.3.7
 splinter>=0.13.0
 -e git+https://github.com/qld-gov-au/ckanext-datarequests@training#egg=ckanext-datarequests
--e git+https://github.com/qld-gov-au/ckanext-data-qld@develop#egg=ckanext-data_qld
+-e git+https://github.com/salsadigitalauorg/ckanext-data-qld@develop#egg=ckanext-data_qld
```
Comment (duttonw) [inline requirements-dev.txt:8]: have we merged the latest ckenext-data-qld into our version yet?

**Comment by duttonw** [comment]
any idea why no circleci build for this?

**Comment by salsa-nathan** [comment]
> any idea why no circleci build for this?

@duttonw not sure sorry - it's building on our setup: https://circleci.com/gh/MarkCalvert/ckanext-ytp-comments/10

**Comment by duttonw** [comment]
found the branch, as its from the feature branch instead of your develop, https://app.circleci.com/pipelines/github/MarkCalvert/ckanext-ytp-comments?branch=feature%2FDQL2-26-QA-remediation as its green, i'm happy for this to be merged.

File: requirements-dev.txt (hunk 1 above)
Comment (duttonw) [reply requirements-dev.txt:8]: tho please change this to qld-gov-au (or we may need to remember to do that post merge)

**Pull request merged**